src/main/java/org/jboss/ejb/protocol/remote/EJBClientChannel.java

```diff
@@ -315,6 +318,12 @@ private void processMessage(final MessageInputStream message) {
                             }
                         }
                         final ServiceRegistration registration = serviceRegistry.registerService(builder.create());
+                        if (Logs.INVOCATION.isDebugEnabled()) {
+                            String moduleLongFormName = distinctName.isEmpty()
```

Comment:
A handy cheat here would be to put a toString method on DiscKey and just log that as an object, removing the need for is*Enabled (and also this might be reusable elsewhere).